README.md

```diff
@@ -381,57 +381,65 @@ Built-in functions. Ready to be bound to any key you like. :smile:
-| `builtin.filetypes`                 | Lists all filetypes.                                                                        |
-| `builtin.highlights`                | Lists all highlights.                                                                       |
-| `builtin.current_buffer_fuzzy_find` | Searches in current buffer lines.                                                           |
-| `builtin.current_buffer_tags`       | Lists Tags in current buffer.                                                               |
-
-### LSP Pickers
-
-| Functions                           | Description                                                                                 |
-|-------------------------------------|---------------------------------------------------------------------------------------------|
-| `builtin.lsp_references`            | Searches in LSP references.                                                                 |
-| `builtin.lsp_document_symbols`      | Searches in LSP Document Symbols in the current document.                                   |
-| `builtin.lsp_workspace_symbols`     | Searches in LSP all workspace symbols.                                                      |
-| `builtin.lsp_code_actions`          | Lists LSP action to be trigged on enter.                                                    |
-| `builtin.lsp_range_code_actions`    | Lists LSP range code action to be trigged on enter.                                         |
-| `builtin.lsp_document_diagnostics`  | Lists LSP Diagnostics in the current document.                                              |
-| `builtin.lsp_workspace_diagnostics` | Lists LSP Diagnostics in the workspace if supported and otherwise open buffers.             |
-| `builtin.lsp_definitions`           | Goto definition if there is only one. If there are multiple, open them up in telescope     |
+| Functions                           | Description                                                                                                                                                 |
+|-------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `builtin.buffers`                   | Lists open buffers in current neovim instance                                                                                                               |
+| `builtin.oldfiles`                  | Lists previously open files                                                                                                                                 |
+| `builtin.commands`                  | Lists available plugin/user commands and runs them on <cr>                                                                                                  |
```

Comment:
don't forget to wrap every `<cr>` with backticks, otherwise it will be treated as an invalid HTML and removed from the rendered markdown.

Reply:
Oh good catch! I'll make sure I revise that before merging.